Aptoide/appcoins-wallet-android
# [MOB-760] Send more error details to rakam

**What does this PR do?**

Adds more details (error code and body) to the logs to rakam

**Database changed?**

No

**Where should the reviewer start?**

OnChainBuyPresenter**.kt**
AppcoinsRewardsBuyPresenter**.kt**

**How should this be manually tested?**

Do a purchase with both credits and APPC (one step and normal) and check if nothing breaks due to the moving from java to kotlin
Try to get a non-403 in both and check if it's logging the event to rakam

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-760


**Code Review Checklist**
